src/js/components/Timestamp.js

```diff
@@ -90,6 +93,7 @@ Timestamp.propTypes = {
     PropTypes.arrayOf(FIELD_TYPES),
     FIELD_TYPES
   ]),
+  seconds: PropTypes.bool,
```

Comment:
I had been wanting to extend the API for Timestamp to allow customizing the fields shown by extending the `fields` component, rather than adding separate properties for different fields. I hadn't worked it out yet though. Perhaps:
```
FIELD_TYPES = ['date', 'time', 'hours', 'minutes', 'seconds']
fields: PropTypes.oneOfType([
  PropTypes.arrayOf(FIELD_TYPES),
  FIELD_TYPES
])
```
where 'time' equates to ['hours', 'minutes']?

Or, perhaps a `precision` property that takes one of `['year', 'month', 'day', 'hour', 'minute', second']`?

Thoughts?